guides/micronaut-oauth2-auth0/micronaut-oauth2-auth0.adoc

```diff
@@ -0,0 +1,123 @@
+include::{commondir}/common-header.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+== OAuth 2.0
+
+To provide authentication, sign in to your https://auth0.com[Auth0] account.
```

Comment:
Use a caret symbol to make the link open in a new window
```suggestion
To provide authentication, sign in to your https://auth0.com[Auth0^] account.
```

Reply:
I don't we we should open links in new windows. I think we should let the reader choose and use the default. Our audience is tech savy. If they want to open links in new windows or tabs they know how to do that. I don't think we should use `target=”_blank”` in our links